solutions/microfrontends/.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": ["docs", "main", "@acme/storybook"]
```

Comment:
I understand the docs and storybook but why do we want to ignore the main app?

Reply:
Because it's not needed to apply versioning to it